src/Types/IDEntitySts.php

```diff
@@ -45,9 +53,13 @@ public static function normalize($value): ?string
      */
     public function isValid(): bool
     {
-        /** @var StsCodeValidatorExtension $validator */
-        $validator = static::getContainer()->make(StsCodeValidatorExtension::class);
+        if (\is_string($this->value) && $this->value !== '') {
+            /** @var StsCodeValidatorExtension $validator */
+            $validator = static::getContainer()->make(StsCodeValidatorExtension::class);
+
+            return \is_string($this->value) && $validator->passes('', $this->value);
```

Comment:
Same. Redundant `\is_string($this->value)`

Reply:
Done